src/alto-ui/Datagrid/components/DatagridCell/__tests__/DataGridCellError.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import DataGridCellError from '../components/DataGridCellError';
+
+describe('DatagridCellError', () => {
+  const showError = jest.fn(val => () => val);
+  const isWarningError = jest.fn(val => () => val);
+  const replaceRowValues = jest.fn().mockReturnValue('Replaced');
+
+  const defaultProps = {
+    column: { key: 1, title: 'title' },
+    row: { 1: '1' },
+    value: 'aa',
+  };
+
+  const getWrapper = ({ ...props }) => shallow(<DataGridCellError {...defaultProps} {...props} />);

```

Comment:
```suggestion
  const getWrapper = props => shallow(<DataGridCellError {...defaultProps} {...props} />);
```
Not work?

Reply:
No because in your approach you can pass only 1st param, and I want to pass multiple arguments so I must use rest operator